week-8/Homework/mandatory/3-dog-photo-gallery/script.js

```diff
@@ -0,0 +1,34 @@
+function onLoad() {
+
+    var butElement = document.getElementById('newDogPhoto');
+    butElement.addEventListener("click", function addDogPhotoToDOM() {
+        fetch('https://dog.ceo/api/breeds/image/random')
+            .then(function(response) {
+                return response.json();
+
+
+            })
+            .then(function(obj) {
+                const listElement = document.createElement('ul');
+                document.body.appendChild(listElement);
+                const imgElement = document.createElement('img');
+                imgElement.src = obj.message;
+                listElement.appendChild(imgElement);
+                imgElement.style.maxWidth = "300px";
+                imgElement.style.maxHeight = "300px";
+                imgElement.style.display = "block";
+                imgElement.style.marginLeft = "auto";
+                imgElement.style.marginRight = "auto";
+
```

Comment:
I'd encourage you to use a class instead of inline style. It's separate the design from the logic.
e.g. `imgElement.classList.add('dog-photo')`